src/components/views/Causes.js

```diff
@@ -15,9 +15,8 @@ class Causes extends Component {
 
   removeCause(id){
     console.log('click', id);
-    socket.emit("causes::remove", id, { cascade: true }, (error, cause) => {
```

Comment:
this event does not execute remove on server (not sure why it works for create and find).